package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "workspaces": [
     "packages/remix-edge-adapter",
+    "packages/remix-serverless-adapter",
```

Comment:
naming (blocking): It's explicit, but we've discussed internally before about the name of this whenever we'd get around to it, and we landed on `@netlify/remix-adapter`